### PR TITLE
Aws test deploy

### PR DESCRIPTION
**Pull request by Shico-UA**
This branch represents interface of the acceptor server.
Please merge this branch to be on the same page with the main PR thread.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -52,7 +52,7 @@ coverage.xml
 
 # Django stuff:
 *.log
-local_settings.py
+acceptor/private.py
```
Comment (auraz) [inline .gitignore]: choose one naming please local_settings.py or private.py

File: .gitignore [hunk 2]
```diff
@@ -52,7 +52,7 @@ coverage.xml
...
 private.py
 
 # Flask stuff:
```
Comment (auraz) [inline .gitignore]: Why we need flask stuff here?

File: graph_creator/templates/graph_creator/graphs.html [hunk 3]
```diff
@@ -1,26 +1,61 @@
...
 
-<div id="instances"></div>
-<div id="courses"></div>
-<div id="students"></div>
+    <h4 class="metrics-header">Metrics</h4>
 
-<script src="{% static 'graph_creator/graphs.js' %}"></script>
-</body>
-</html>
+    <div class="container metrics-container">
```
Comment (auraz) [inline graph_creator/templates/graph_creator/graphs.html]: where are bootstrap tables?

File: static/graph_creator/worldmap.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+                // Tooltip content
+                return ['<div class="hoverinfo">',
+                    '<strong>', geo.properties.name, '</strong>',
+                    '<br>Students: <strong>', data.numberOfThings, '</strong>',
+                    '</div>'].join('');
+            }
+        }
+    });
+}());
```
Comment (auraz) [inline static/graph_creator/worldmap.js]: no newline in the end of file

File: .gitignore (hunk 1 above)
Comment (Shico-UA) [inline .gitignore]: We use private.

File: .gitignore (hunk 2 above)
Comment (Shico-UA) [inline .gitignore]: Removed. We don'n need it, this is just "standard" python .gitignore template.

File: graph_creator/templates/graph_creator/graphs.html (hunk 3 above)
Comment (Shico-UA) [inline graph_creator/templates/graph_creator/graphs.html]: Bootstrap table is inside of graph_creator/templates/graph_creator/worldmap.html file.
Or what do you mean?

File: static/graph_creator/worldmap.js (hunk 4 above)
Comment (Shico-UA) [inline static/graph_creator/worldmap.js]: Done

File: graph_creator/fixtures/graph_creator.json [hunk 5]
```diff
@@ -169,14 +169,14 @@
...
       "platform_name": null,
       "platform_url": null,
       "secret_token": "platform3",
       "statistics_level": "1",
-      "students_per_country": "{\"RU\": 4523, \"DE\": 10034, \"FR\": 203, \"null\": 1}"
+      "students_per_country": "{\"FR\": 5232, \"DE\": 4251, \"FR\": 3422, \"null\": 1}"
     }
   }
-]
+]
```
Comment (DmytryiStriletskyi) [inline graph_creator/fixtures/graph_creator.json]: No newline at the end.

Comment (Shico-UA) [inline graph_creator/fixtures/graph_creator.json]: Done

**Pull request merged**